CHANGELOG.md

```diff
@@ -71,13 +71,13 @@ Version 4.0.0 of the IDS Information Model
 * Redesign of `ids:Endpoint` class.
   * Removed `ids:InteractiveEndpoint`.
   * `ids:StaticEndpoint` is now called `ids:ConnectorEndpoint`. `ids:ConnectorEndpoint` contains additional properties, especially for endpoint description and documentation.
-  * `ids:Host` information is now part of the `ids:Endpoint`.
+  * `ids:Endpoint` information is now part of the `ids:Endpoint`.
```

Comment:
```suggestion
  * `ids:Host` information is now part of the `ids:Endpoint`.
```

Ahm I think this was actually correct. The ids:Host was removed and its information was integrated into ids:Endpoint. Please revert (same for line 74 in this file), thank you!

Reply:
Yes, you are right. This is a mistake while replacing all.